html/index.html

```diff
@@ -0,0 +1,713 @@
+          alt="started-background"
+          width="1440"
+          height="406"
+        /> -->
+        <div class="started-block____bg"></div>
+        <div class="started-block____left">
+          <p class="started-block____left-title">IT’S FAST, FREE AND VERY EASY!</p>
+          <h2 class="block-title">Ready to get started?</h2>
+          <div>
+            <p>
+              Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque
+            </p>
+            <p>
+              Laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+              architecto beatae vitae dicta sunt explicabo.
+            </p>
+          </div>
+          <a class="started-block____left-apply-btn button" href="#form">Apply Now</a>
+          <button class="started-block____left-request-btn button">Request Service</button>
+        </div>
+        <div class="started-block____right">
+          <div class="started-block____right-title">
+            <svg class="chain-icon" width="24" height="24">
+              <use xlink:href="/images/sprite.svg#chain"></use>
+            </svg>
+            <h2 class="block-title">Quick Link</h2>
+          </div>
+          <div class="started-block____right-box">
+            <div class="started-block____right-box-item">
+              <svg width="24" height="24">
+                <use xlink:href="/images/sprite.svg#pen"></use>
+              </svg>
+              <h4>Tution and Fees</h4>
+            </div>
+            <div class="started-block____right-underline"></div>
+            <div class="started-block____right-box-item">
+              <svg width="24" height="24">
+                <use xlink:href="/images/sprite.svg#case"></use>
+              </svg>
+              <h4>Review & Rating</h4>
+            </div>
+            <div class="started-block____right-underline"></div>
+
+            <div class="started-block____right-box-item">
+              <svg width="24" height="24">
+                <use xlink:href="/images/sprite.svg#book"></use>
+              </svg>
+              <h4>Universoty Facilities</h4>
+            </div>
+            <div class="started-block____right-underline"></div>
+
+            <div class="started-block____right-box-item">
+              <svg width="18" height="18">
+                <use xlink:href="/images/sprite.svg#message"></use>
+              </svg>
+              <h4>Community Q&A</h4>
+            </div>
+            <div class="started-block____right-underline"></div>
+          </div>
+        </div>
+      </section>
+
+      <section class="blog-block">
+        <div class="blog-block____title">
+          <h2 class="block-title">Latest Blog</h2>
+          <div class="title-underline"></div>
+        </div>
+        <div class="blog-block____box">
+          <div class="blog-block____box-item">
+            <div class="blog-block____box-item-background"></div>
```

Comment:
Same problem. You use separate div for background instead of set it to main container of this card. Then you have to set property right with negative value instead of 0. Also hardcoded position of every element of this card. Try place some element to container.

Reply:
Done.